openquake/server/db/actions.py

```diff
@@ -478,12 +478,16 @@ def calc_info(db, calc_id):
     :returns: dictionary of info about the given calculation
     """
     job = db('SELECT * FROM job WHERE id=?x', calc_id, one=True)
+    fields_to_exclude = ['ds_calc_dir', 'pid']
```

Comment:
I would also exclude the field `relevant`

Reply:
We finally decided to keep it